omnisafe/algos/on_policy/cppo_pid.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-
+"""Implementation of the CPPOPid algorithm."""
```

Comment:
Please add an extra linebreak between the module docstring and imports. Same changes for other source files.